chatterbot/utils/queues.py

```diff
@@ -0,0 +1,30 @@
+class ResponseQueue(object):
```

Comment:
The name if this class may be misleading. It's more of a list. It needs to possible to check if an object is in the list, and it needs to have a size limit in such a way that the oldest object is removed when adding a new object. I'm not sure if overriding queue has any advantages in this case.